### PR TITLE
Add GitHub actions

### PR DESCRIPTION
**Pull request by wannaphong**
I add GitHub actions for test this project.

### CODE REVIEW

File: .github/workflows/python-test.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6]
```
Comment (p16i) [inline .github/workflows/python-test.yml]: should we have more python versions here?

File: .github/workflows/python-test.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name: Test Python Package
```
Comment (p16i) [inline .github/workflows/python-test.yml]: I would use something like "Unit Tests" .

**Comment by wannaphong** [comment]
@heytitle  OK. I updated it.

File: .github/workflows/python-test.yml (hunk 1 above)
Comment (wannaphong) [reply .github/workflows/python-test.yml]: I want to add 3.6 - 3.9.

File: .github/workflows/python-test.yml (hunk 2 above)
Comment (wannaphong) [reply .github/workflows/python-test.yml]: ok. I am rename it.

**Comment by wannaphong** [comment]
I remove python 3.9 because numpy have not build for python 3.9.

**Pull request merged**